rdmo/questions/models/page.py

```diff
@@ -235,6 +235,12 @@ def descendants(self):
             descendants += [element, *element.descendants]
         return descendants
 
+    @cached_property
+    def attributes(self):
+        attributes = [self.attribute] if self.attribute else []
+        attributes += [descendant.attribute for descendant in self.descendants if descendant.attribute]
```

Comment:
Is it not more readable just like this? standard notation for list comprehension..
```py
attributes += [i.attribute for i in self.descendants if i.attribute]
```